src/azure-cli-core/azure/cli/core/local_context.py

```diff
@@ -0,0 +1,101 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import stat
+import shutil
+from six.moves import configparser
```

Comment:
We don't need `six` anymore as Python 2 is already dropped.

Reply:
updated